### PR TITLE
docs: fix badges (fixes #141)

### LINKED ISSUE #141: travis badge in readme broken
Clicking on the Travis badge in the readme take you nowhere.

### PR DESCRIPTION
**Pull request by g-plane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/cherow/cherow)
-[![Build status](https://ci.appveyor.com/api/projects/status/pkydnkv0deckns5l/branch/master?svg=true)](https://ci.appveyor.com/project/Kflash/cherow/branch/master)
-[![CircleCI](https://circleci.com/gh/cherow/cherow.svg?style=svg)](https://circleci.com/gh/cherow/cherow)
-[![build status](https://api.travis-ci.org/cherow/cherow.svg?branch=master)](https://api.travis-ci.org/cherow/cherow)
-[![Coverage Status](https://coveralls.io/repos/github/cherow/cherow/badge.svg?branch=master)](https://coveralls.io/github/cherow/cherow?branch=master)
+[![NPM version](https://img.shields.io/npm/v/cherow.svg?style=flat-square)](https://www.npmjs.com/package/cherow)
+[![Gitter chat](https://img.shields.io/gitter/room/Cherow/cherow.svg?style=flat-square)](https://gitter.im/cherow/cherow)
+[![Build status](https://img.shields.io/appveyor/ci/Kflash/cherow/master.svg?style=flat-square&logo=appveyor)](https://ci.appveyor.com/project/Kflash/cherow/branch/master)
+[![CircleCI](https://img.shields.io/circleci/project/github/cherow/cherow/master.svg?style=flat-square)](https://circleci.com/gh/cherow/cherow)
```
[the remark this answers is not in the document]
Comment (Aladdin-ADD) [reply README.md]: the badges show the status of master, but target to someone else -- that's weird.

[the remark this answers is not in the document]
Comment (KFlash) [reply README.md]: I guess so

[the remark this answers is not in the document]
Comment (g-plane) [reply README.md]: So should I remove the branch info in those links?

[the remark this answers is not in the document]
Comment (Aladdin-ADD) [reply README.md]: I prefer targeting to branch master, so the link should be something like `https://travis-ci.org/cherow/cherow/branch/master`.

[the remark this answers is not in the document]
Comment (g-plane) [reply README.md]: However it seems there is no such branch-specified link in Travis CI.

[the remark this answers is not in the document]
Comment (Aladdin-ADD) [reply README.md]: emm.. I see.  then I'm fine to merge like this. 😄

**Pull request merged**